A42-xds-ring-hash-lb-policy.md

```diff
@@ -452,16 +452,19 @@ Another issue is the way that the `priority` policy handles its failover
 timer.  The failover timer is used to apply an upper bound to the amount
 of time that `priority` policy waits for a child policy to become
 connected before it gives up and creates the child policy for the next
-priority.  The failover timer is started when the child reports state
-`CONNECTING` and is cancelled when the child reports any other state.
+priority.  The failover timer is started when a child is first created
+and is cancelled when the child reports any state other than `CONNECTING`.
 To allow this timer to work properly, the `ring_hash` policy should
 remain in state `CONNECTING` until it transitions to either
 `TRANSIENT_FAILURE` or `READY`.  And, just as in the `TRANSIENT_FAILURE`
 case above, it will proactively attempt to connect to at least one
 subchannel at all times while it is reporting `CONNECTING`, so that it
 does not stay in state `CONNECTING` indefinitely if it is not receiving
 picks (e.g., if the application is only occassionally starting RPCs and
-giving them very short deadlines).
+giving them very short deadlines).  Additionally, we will change the
```

Comment:
We don't talk about why we are doing this, and it is subtle. Maybe add a "This is why we changed the priority failover time logic" in the "Note that when the `ring_hash` policy first starts up with a completely" paragraph?